src/editor.js

```diff
@@ -177,8 +177,7 @@ export class OdooEditor extends EventTarget {
         this._collaborate = false;
         this._collaborate_last = null;
 
-        // used to check if we have to rollback an operation as an unbreakable is
-        this._torollback = false; // unbreakable removed or added
+        this._torollback = false;
```

Comment:
Ok il était incomplet car unbreakable n'est pas le seul élément qui pouvait modifier cette variable.
Je remplace par: 
> Track if we need to rollback mutations in case unbreakable or unremovable are being added or removed.